### PR TITLE
Added styled component to AddToCollectionList component

### PR DESCRIPTION
**Pull request by adadgarg**
Changes:
Added styled component to AddToCollectionList component.

I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] has no test errors (`npm run test`)
* [x] is from a uniquely-named feature branch and is up to date with the  `develop` branch.
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

**Comment by adadgarg** [comment]
Hi @lindapaiste, I didn't see your PR #2353 before making this PR. I will make the changes later when you will merge the new code of AddCollectionList and AddCollectionSketchList components. We can put this as a potential fix as these styled components can be used in the new code.

**Comment by lindapaiste** [comment]
@adityagarg06 Sure thing.  We can merge either my PR or yours first and then fix the conflicts that show up.  Realistically your PR gets merged first.

We have two components which are really similar (and it's confusing which is which!) but they both use these same two CSS classes which you converted.  Maybe we do some sort of shared wrapper at some point, but for now I think you should `export` your two styled components and also use them in the [`AddToCollectionSketchList.jsx`](https://github.com/processing/p5.js-web-editor/blob/develop/client/modules/IDE/components/AddToCollectionSketchList.jsx).  Then you can delete those classes from the CSS.

If we want to make an improvement to the CSS while we're in here, how about we change it from:
```
min-width: ${remSize(600)};
```
to:
```
width: ${remSize(600)};
max-width: 100%;
```
That will help a lot with responsiveness.

File: client/modules/IDE/components/AddToCollectionList.jsx [hunk 1]
```diff
@@ -13,10 +13,26 @@ import * as SortingActions from '../actions/sorting';
 import getSortedCollections from '../selectors/collections';
 import Loader from '../../App/components/loader';
 import QuickAddList from './QuickAddList';
+import { remSize } from '../../../theme';
 
 const projectInCollection = (project, collection) =>
   collection.items.find((item) => item.projectId === project.id) != null;
 
+const CollectionAddSketchWrapper = styled.div`
```
Comment (lindapaiste) [inline client/modules/IDE/components/AddToCollectionList.jsx]:
```suggestion
export const CollectionAddSketchWrapper = styled.div`
```

File: client/modules/IDE/components/AddToCollectionList.jsx [hunk 2]
```diff
@@ -13,10 +13,26 @@ import * as SortingActions from '../actions/sorting';
...
+  &&& {
+    min-width: ${remSize(600)};
```
Comment (lindapaiste) [inline client/modules/IDE/components/AddToCollectionList.jsx]:
```suggestion
    width: ${remSize(600)};
    max-width: 100%;
```

File: client/modules/IDE/components/AddToCollectionList.jsx [hunk 3]
```diff
@@ -13,10 +13,26 @@ import * as SortingActions from '../actions/sorting';
 import getSortedCollections from '../selectors/collections';
 import Loader from '../../App/components/loader';
 import QuickAddList from './QuickAddList';
+import { remSize } from '../../../theme';
 
 const projectInCollection = (project, collection) =>
   collection.items.find((item) => item.projectId === project.id) != null;
 
+const CollectionAddSketchWrapper = styled.div`
+  &&& {
```
Comment (lindapaiste) [inline client/modules/IDE/components/AddToCollectionList.jsx]: It doesn't really matter but I suspect that you'd be fine without the `&&&`?

File: client/modules/IDE/components/AddToCollectionList.jsx [hunk 4]
```diff
@@ -13,10 +13,26 @@ import * as SortingActions from '../actions/sorting';
...
+    min-width: ${remSize(600)};
+    overflow: auto;
+  }
+`;
+
+const QuickAddWrapper = styled.div`
+  &&& {
+    min-width: ${remSize(600)};
```
Comment (lindapaiste) [inline client/modules/IDE/components/AddToCollectionList.jsx]:
```suggestion
    width: ${remSize(600)};
    max-width: 100%;
```

**Review by lindapaiste — CHANGES_REQUESTED** [review]
I'm playing around with the github suggestions tool!

**Review by lindapaiste — APPROVED** [review]
Perfect! Thank you.

**Pull request merged**